msal_extensions/cache_lock.py

```diff
@@ -25,7 +28,24 @@ def __init__(self, lockfile_path):
             flags=portalocker.LOCK_EX | portalocker.LOCK_NB,
             **open_kwargs)
 
+    def try_to_create_lock_file(self):
+        retries_no = 100
+        retry_delay_milliseconds = 0.150
```

Comment:
Do we have a reasoning for choosing this pair of numbers? What did our Java friends use here?
FWIW, portalocker's default settings is 0.25 interval, and 5 seconds waiting maximum. (BTW, I also like the way they control this by maximum waiting time, rather than by retries_no.)

Reply:
I need to verify these values from Java team. I dont know if these are good defaults. I have pinged Petro to go through the PR but I guess I can ask him what defaults have been used in Java. From what I recollect its 6 secs waiting time not sure what the interval is. I agree we can control this by maximum waiting time rather than retries_no. I will make that change after verifying the  default timeout values.I think portalocker's default settings look reasonable to me.